py/server/deephaven/table.py

```diff
@@ -319,6 +320,34 @@ def meta_table(self) -> Table:
     def j_object(self) -> jpy.JType:
         return self.j_table
 
+    def get_table_attributes(self) -> Dict[str, Any]:
+        """Returns all the attributes defined on the table."""
+        j_map = jpy.cast(self.j_table, _JLiveAttributeMap).getAttributes()
+        return j_map_to_dict(j_map)
```

Comment:
We are bleeding java objects!  `getAttributes()` returns `Map<String,Object>`, and `j_map_to_dict` converts to a dict without wrapping the result types.

1. What are the attribute types?
2. How do we handle them in Python without bleeding Java objects?

Reply:
1. Even though from the current usages, the types of attributes are only primitive ones and strings, it does leave doors open for more complex objects, so type hint [Any] is appropriate.
2. We'll wrap the Java objects (non primitive/strings) if a wrapper exists, otherwise it will just be JPY wrapped ones and should be used as such. But most likely if it is something important for the users we should already have wrapped it in Python.